vllm/v1/engine/core_client.py

```diff
@@ -453,6 +457,9 @@ def _init_core_engines(
     def shutdown(self):
         self._finalizer()
 
+    def get_gpu_blocks(self) -> int:
+        return self.vllm_config.cache_config.num_gpu_blocks
```

Comment:
This shouldn't be needed - the logger should just be able to read from `VllmConfig`

Reply:
this change will return VllmConfig.